packages/web/README.md

```diff
@@ -1 +1,41 @@
-# Algob Web
+# Web
+
+Web allows you to interact with contracts easily. It is designed to be used with web dapps.
+
+This package provides a class `WebMode` which has variety of high level functions like, `waitForConfirmation`, `executeTransaction`, `signTransaction`, etc. These functions help in sending transactions easier when building dapps using react.
+
+It is not possible to use `algob` directly in a react app, because `algob` uses nodejs file system which cannot be used in a react app. Therefore we created a lightweight packake web which can be used to create and send transactions to the network from react app.
```

Comment:
```suggestion
### Relation to algob

`algob` uses `@algo-builder/web` package. However It is not possible to use `algob` directly in a web app, because `algob` uses nodejs file system. Therefore we created a lightweight `@algo-builder/web` package to provide common functionality and support dapp development.
```